lib/validator.ex

```diff
@@ -6,4 +6,17 @@ defmodule Validator do
       _ -> false
     end
   end
+
+  def convert_input(user_location) when elem(user_location, 0) == true,
+    do: String.to_integer(elem(user_location, 1))
+
+  def is_numerical_location(input) do
+    {String.match?(input, ~r/^[0-9]*$/), input}
+  end
```

Comment:
I don't know if this is working correctly. I can't seem to input symbols without the game breaking.